client/src/Routing.js

```diff
@@ -20,6 +22,12 @@ const Routing = () => {
             <Route path="/statistics" element={<AuthenticationGuardRoute />}>
                 <Route path="/statistics" element={<Statistics />} />
             </Route>
+            <Route path="/profile/followers" element={<AuthenticationGuardRoute />}>
```

Comment:
Remember to update when Shuker adds tabbar if she merges before you
Also, it should have been `/user/:email/followers` and `/user/:email/followings`

Reply:
אם אתה מתכוון בפרונט סבבה, 
בבקאנד זה מוגדר עם follows ולא user

Reply:
כן הכוונה ל-route בפרונט